integration-tests/contract-security-zero-transfer.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { CONFIGS } from './config';
+import { RpcClient } from '@taquito/rpc';
+
+// TC-007 - A 0tez transaction to an implicit account should fail.
+
+const client = new RpcClient(' https://ithacanet.ecadinfra.com');
+
+CONFIGS().forEach(({ lib, rpc, setup }) => {
+  const Tezos = lib;
+
+  describe(`Test contracts using: ${rpc}`, () => {
+    beforeEach(async (done) => {
+      await setup();
+
+      done();
+    });
+
+    it('Verify that Transactions of 0ꜩ towards a contract without code are forbidden', async () => {
+      const op = await Tezos.contract.originate({
```

Comment:
Can you explain why there is an origination first in the test?
It is not used in the "second part" (const opTransfer = await Tezos.contract.transfer({ to: publicKeyHash, amount: 0 });) and doesn't seem to be related.